experiments/TestCase/baroclinic_wave_fvm.jl

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env julia --project
+
+using ArgParse
+using LinearAlgebra
+using StaticArrays
+using Test
+
+using ClimateMachine
+using ClimateMachine.Atmos
+using ClimateMachine.Orientations
+using ClimateMachine.ConfigTypes
+using ClimateMachine.Diagnostics
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.SystemSolvers: ManyColumnLU
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.TemperatureProfiles
+using ClimateMachine.Thermodynamics:
+    air_density, air_temperature, total_energy, internal_energy, PhasePartition
+using ClimateMachine.TurbulenceClosures
+using ClimateMachine.VariableTemplates
+import ClimateMachine.DGMethods.FVReconstructions: FVLinear
+
+using CLIMAParameters
+using CLIMAParameters.Planet: MSLP, R_d, day, grav, Omega, planet_radius
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+function init_baroclinic_wave!(problem, bl, state, aux, localgeo, t)
+    FT = eltype(state)
+
+    # parameters
```

Comment:
Not a big deal, but in the majority of comments style, they start with a capital letter. I would capitalize them for consistency.